cli/src/pcluster/cli/commands/cluster.py

```diff
@@ -37,6 +37,26 @@
 LOGGER = logging.getLogger(__name__)
 
 
+class VersionCommand(CliCommand):
```

Comment:
I was thinking that all of these would move as a separate pass and just have some `commands.py` that defines the non-api operations (or some organization beyond that)... since I don't think we'll have any "cluster commands" once we convert over. It can be moved now as an initial effort.